packages/sampler/stories/default/Platform.js

```diff
@@ -3,32 +3,35 @@ import platform from '@enact/core/platform';
 import webosPlatform from '@enact/webos/platform';
 import {storiesOf} from '@storybook/react';
 import {withInfo} from '@storybook/addon-info';
+import {Scroller} from '@enact/ui/Scroller';
 
-const {gesture, platformName, touch} = platform;
-const version = platform[platformName];
-
-const deviceType = () => {
-	let device = '';
-	for (let p in webosPlatform) {
-		if (webosPlatform[p]) {
-			device = p;
-		}
-	}
-	return device;
-};
+function logObject (object) {
+	return Object.keys(object)
+		.filter(key => object[key] != null)
+		.map(key => {
+			let value = object[key];
+			if (value === false) {
+				value = 'false';
+			} else if (value === true) {
+				value = 'true';
+			}
+			return <div key={key}>{key}: {value}</div>;
+		});
+}
 
 storiesOf('Core', module)
 	.add(
 		'Platform',
 		withInfo('Detection')(() => (
-			<div>
-				Detected: {platformName} {platform.webos ? `(${deviceType()})` : ''}
-				<br />
-				Version: {version}
-				<br />
-				Gesture support: {gesture ? 'true' : 'false'}
-				<br />
-				Touch support: {touch ? 'true' : 'false'}
-			</div>
+			<Scroller>
+				<h3>
+					Platform:
```

Comment:
Do these have to be on their own line, or can they just be in one line? `<h3>Platform:</h3>`

Reply:
I don't think it matters to me.  If you feel strongly, we can change it!

Reply:
Oh boy yes. It's like the tag exploded. Please change this.

Reply:
I did.